app/ui/client/views/app/room.js

```diff
@@ -883,7 +883,7 @@ Template.room.events({
 		const { msg } = messageArgs(this);
 		const link = msg.attachments ? msg.attachments[0].message_link : this.message_link;
 		const repliedMessageId = link.split('?msg=')[1];
-		const { pathname } = new URL(link);
+		const pathname = `/${ link.split('?msg=')[0].split('/').slice(-2)[0] }/${ link.split('?msg=')[0].split('/').slice(-1)[0] }`;
```

Comment:
@bhardwajaditya This is not actually the fix that was merged into our develop_pwa branch...  please see:  https://github.com/WideChat/Rocket.Chat/pull/226/files

Reply:
Ohh I am sorry. I mixed up the commit.